editor/modes/visual-editor/block.js

```diff
@@ -232,7 +232,7 @@ class VisualEditorBlock extends wp.element.Component {
 				tabIndex="0"
 				{ ...wrapperProps }
 			>
-				{ ( showUI || isHovered ) && <BlockMover uids={ [ block.uid ] } /> }
+				{ ( showUI || isHovered ) && <BlockMover uid={ block.uid } order={ this.props.order } /> }
```

Comment:
Ah, thank you for spotting and apologies for not seeing it myself! I'll look into amending this tonight and allowing for the case of multiple blocks. Likely the string will just have to read something like "Move _n_ blocks {down|up} by one place".